hwangsae/transmuxer.c

```diff
@@ -331,9 +331,9 @@ hwangsae_transmuxer_parse_segments (HwangsaeTransmuxer * self,
 
   /* Make base times start from zero. */
   for (it = g_list_last (segments); it; it = it->prev) {
+    ((Segment *) it->data)->end_time -= ((Segment *) segments->data)->base_time;
```

Comment:
good catch!

Reply:
The more we test the more bugs we found!